src/Helper/AttributesHelper.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * @see       https://github.com/laminas/laminas-view for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-view/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-view/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Laminas\View\Helper;
+
+use Laminas\View\Attributes;
+use Traversable;
+
+/**
+ * Helper for creating Attributes objects
+ */
+class AttributesHelper extends AbstractHelper
```

Comment:
We have the following helpers:

* [HtmlTag](https://docs.laminas.dev/laminas-view/helpers/html-tag/)
* [HtmlList](https://docs.laminas.dev/laminas-view/helpers/html-list/)
* [HtmlObject](https://docs.laminas.dev/laminas-view/helpers/html-object/)
* …

And this helper creates and HTML attributes therefore the name should also contain the prefix "Html".
With the [support of autocompletion](https://github.com/laminas/laminas-view/blob/c34b15f42cac410f35010163de169ab71c39bf73/src/Renderer/PhpRenderer.php#L25-L75) in IDEs, the length of the name is not a major problem.

Reply:
The new helper must be added to the DocBlock of [`Laminas\View\Renderer\PhpRenderer`](https://github.com/laminas/laminas-view/blob/c34b15f42cac410f35010163de169ab71c39bf73/src/Renderer/PhpRenderer.php#L75).

Reply:
@froschdesign Should `Attributes` also be renamed to `HtmlAttributes`? If not, maybe the `Helper` suffix could be dropped, since it is not used in other helpers apart from the abstract base class? I only added the suffix to make the helper class name distinct from the `Attributes` class.

Reply:
@aleksip 
> …the Helper suffix could be dropped…

Correct. Use `HtmlAttributes` as class name for the helper and I think we can use `HtmlAttributesSet` for the current `Attributes` class.